recipes/_service.rb

```diff
@@ -3,6 +3,7 @@
 template node['kibana']['service']['template_file'] do
   cookbook node['kibana']['service']['cookbook']
   source node['kibana']['service']['source']
+  mode '0755'
```

Comment:
Can you make this -
mode 0o0755 
to satisfy more current version of rubocop runs?